### DISCUSSION TITLE
Blocking visibility of work files/directories in logical view, locally or globally?

### DISCUSSION THREAD

**Discussion by DKnoto** [albilu/netbeansPython#94] [Ideas]
[Continuation of submission #90](https://github.com/albilu/netbeansPython/issues/90)

> > 2. In sections Sources and Tests directory **pycache** should not be visible;
> 
> You can configure wich files/directory you want to hide in Tools -> Options -> Miscellaneous -> Ignored Files Pattern
> 

First of all, the idea of global locking is dangerous because it can interfere with projects in other languages.

Secondly, the solution you recommended unfortunately did not work as expected. Only one __pycache__ directory out of two disappeared:

![No-90__pycache__A-c128](https://github.com/albilu/netbeansPython/assets/1998807/a72c4b2a-5dd4-46d1-81b1-891e59572468)

![No-90__pycache__B-c064](https://github.com/albilu/netbeansPython/assets/1998807/c0f2f203-cef9-4628-928b-9869b24bca39)

**Comment by albilu** [comment]
> Secondly, the solution you recommended unfortunately did not work as expected. Only one **pycache** directory out of two disappeared:

Right there is an issue with some subdirectories
